doc/openapi.yaml

```diff
@@ -0,0 +1,117 @@
+openapi: 3.0.1
```

Comment:
At this point, the hub supports only JSON files, that live in `doc` and are named 'rest.json', 'restapi.json', 'swaggers.json', or 'swagger.json'.
This is very likely to change, but now there is that limitation.